CHANGELOG.md

```diff
@@ -1,3 +1,14 @@
+
+v1.12.1 / 2022-12-15
+==================
+
+  * `ObjectCompletionProviderd` should not provide completion items that don't live in the `SourceIndex` ([#690](https://github.com/Shopify/theme-check/issues/690))
```

Comment:
Can we fix the typo in `ObjectCompletionProviderd`? :D
I know, my fault, didn't check the PR title x)

Reply:
good catch! yes